start-ui.bat

```diff
@@ -12,9 +12,11 @@ goto :end
 :activate_venv
 echo activating venv %VENV_DIR%
 set PYTHON="%VENV_DIR%\Scripts\python.exe"
+if defined PROFILE (set PYTHON=%PYTHON% -m scalene --off --cpu --gpu --profile-all --no-browser)
```

Comment:
I don't see any output when running the profiler. Do you still need to configure that? Also, what does the --off parameter? I can't see that in the documentation.

Reply:
You need to go into the profiling window and click "Start Profiling" when you want it to start and, later, "Stop Profiling". The --off flag tells it to only turn on when the code tells it to turn on, rather than at launch.

Once the process exits, it will write "profile.html".